common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -85,6 +85,16 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
+  val AdomikSwitch = Switch(
+    SwitchGroup.Commercial,
+    "adomik",
+    "Enable Adomik traffic splitting.",
+    owners = Seq(Owner.withName("commercial team")),
+    safeState = Off,
+    sellByDate = new LocalDate(2017, 8, 30),
```

Comment:
I think they want to run for at least one full calendar month, so maybe push this back to the first week of October?